### DISCUSSION TITLE
How to create a New Odoo Instance [v15CE] on to Prod Server with [v14CE]

### DISCUSSION THREAD

**Discussion by luismata01** [Tecnativa/doodba#498] [Doodba - The template]
Hi Team

How to or what is the recommendation to create a New Odoo Instance [v15CE] on to Prod Server with [v14CE], in this case this work with Traefik,. 

I Need this to migrate [v14CE] to [v15CE]

Thank's a Lot

Luis

**Comment by pedrobaeza** [comment]
There's no way right now to migrate from v14 to v15 with OpenUpgrade.
